fedoracommunity/connectors/bodhiconnector.py

```diff
@@ -525,13 +525,28 @@ def query_active_releases(self, filters=None, **params):
             filters = dict()
         filters = self._query_updates_filter.filter(filters, conn=self)
         package = filters.get('package')
-        pkgdb = get_connector('pkgdb')
         koji = get_connector('koji')._koji_client
         koji.multicall = True
 
-        for release in pkgdb.get_fedora_releases():
-            tag = release[0]
-            name = release[1]
+        releases_obj = self.call('releases', {})
+        total_pages = releases_obj['pages']
+        releases_all = releases_obj['releases']
```

Comment:
It seems like this would work but I think you could tighten this up with (or something better):

```python
releases_obj = self.call('releases', {})
releases_all = None
for i in range(1, releases_obj['pages'] + 1):
    if releases_all == None:
        releases_all = releases_obj['releases']
        continue
    temp = self.call('releases?page='+str(i), {})['releases']
    releases_all.extend(temp) 
```